File: internal/domain/entities/storage_drives_types.go
package entities

type StorageDrivesTypes struct {
	ID          []string `sql:"primary_key" json:"id,omitempty"`
	Description []string `json:"description,omitempty"`
}

type storageDrivesTypesRepository struct {
	RepositoryName string

	ID          string
	Description string
}

func StorageDrivesTypesRepository() storageDrivesTypesRepository {
	return storageDrivesTypesRepository{
		RepositoryName: "storage_drives_types",

		ID:          "id",
		Description: "description",
	}
}

File: web/src/domain/entities/storage_drives_types.ts
namespace _StorageDrivesTypes {
	export enum FieldColumn {
		ID = 'id',
		Description = 'description'
	}

	export const RepositoryName = 'storage_drives_types'

	export interface Interface {
		id?: string[]
		description?: string[]
		storage_drive_type?: any[]
	}
}

export default _StorageDrivesTypes
